setup.py

```diff
@@ -39,7 +39,7 @@
       scripts=glob('scripts/*py'),
       setup_requires=['numpy >= 1.9.2'],
       install_requires=[
-          'IPython < 4.0.0',
+          'IPython',
```

Comment:
The editing capabilities were broken when I tried back in October, and I
thought you had indicated so as well? Will test when on my laptop next
On Dec 10, 2015 2:30 PM, "J W Debelius" <notifications@github.com> wrote:

> In setup.py
> <https://github.com/biocore/American-Gut/pull/182#discussion_r47296594>:
>
> > @@ -39,7 +39,7 @@
> >        scripts=glob('scripts/*py'),
> >        setup_requires=['numpy >= 1.9.2'],
> >        install_requires=[
> > -          'IPython < 4.0.0',
> > +          'IPython',
>
> Ive only ever run ipymd with IPython 4.0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/182/files#r47296594>.
>